codecept.conf.js

```diff
@@ -9,8 +9,9 @@ module.exports = {
   name: 'right-to-rent-check',
   tests: './apps/**/acceptance/features/*.js',
   include: {
+    tenantInUKPage: pagesPath('tenant-in-UK.js'),
```

Comment:
ahh ok, I was just following what was asked for in the url for the content.  I'll change that